qiskit_ibm_runtime/runtime_options.py

```diff
@@ -70,7 +70,12 @@ def __init__(
                 this time limit, it is forcibly cancelled. Simulator jobs continue to use wall
                 clock time.
             session_time: Length of session in seconds.
-            private: Boolean of whether or not the job is marked as private.
+            private: Boolean that indicates whether the job is marked as private. When set to true,
+                input parameters are not returned, and the result can only be read once; after it is
+                read, both the input parameters and the results are deleted from the service. If the
+                result is not read, the parameters and results are automatically deleted after a
+                specified time. When set to false, the input parameters and results follow the standard
+                retention behavior.
```

Comment:
Should we also mention that it's not supported on cloud ([comment](https://github.com/Qiskit/qiskit-ibm-runtime/pull/1888#issuecomment-2512949360))?

Reply:
Nit: to be exact, which may not matter in this context: input parameters are deleted when the job is terminated. The result is deleted after it is read